include/sapi/tss2_tcti.h

```diff
@@ -146,6 +151,14 @@ typedef TSS2_RC (*TSS2_TCTI_GET_POLL_HANDLES_FCN) (
 typedef TSS2_RC (*TSS2_TCTI_SET_LOCALITY_FCN) (
     TSS2_TCTI_CONTEXT *tctiContext,
     uint8_t locality);
+typedef TSS2_RC (*TSS2_TCTI_MAKE_STICKY_FCN) (
+    TSS2_TCTI_CONTEXT *tctiContext,
+    TPM2_HANDLE *handle,
+    uint8_t sticky);
+typedef TSS2_RC (*TSS2_TCTI_INIT_FUNC) (
```

Comment:
Should the pointer name be consistent with the above i.e. *_FCN instead of _FUNC